src/lets-version.js

```diff
@@ -612,32 +619,34 @@ export async function applyRecommendedBumpsByPackage(opts) {
   }
 
   // commit the stuffs
-  const header = 'Version Bump';
-  const body = synchronized.bumps.map(b => `${b.packageInfo.name}@${b.to}`).join(os.EOL);
-  if (dryRun) {
-    console.info(
-      `~~~~~${os.EOL}Will create a git commit with the following message:${os.EOL}${os.EOL}${header}${os.EOL}${os.EOL}${body}${os.EOL}~~~~~`,
-    );
-  } else await gitCommit(header, body, '', fixedCWD);
+  if (!noCommit) {
+    const header = 'Version Bump';
+    const body = synchronized.bumps.map(b => `${b.packageInfo.name}@${b.to}`).join(os.EOL);
+    if (dryRun) {
+      console.info(
+        `~~~~~${os.EOL}Will create a git commit with the following message:${os.EOL}${os.EOL}${header}${os.EOL}${os.EOL}${body}${os.EOL}~~~~~`,
+      );
+    } else await gitCommit(header, body, '', fixedCWD);
+  }
 
   // create all the git tags
-  const tagsToPush = await Promise.all(
-    synchronized.bumps.map(async b => {
-      const tag = formatVersionTagForPackage(
-        new PackageInfo({
-          ...b.packageInfo,
-          version: b.to,
-        }),
-      );
-      if (dryRun) console.info(`Will create the following git tag: ${tag}`);
-      else await gitTag(tag, fixedCWD);
+  if (!noPush || !noCommit) {
+    const tagsToPush = await Promise.all(
```

Comment:
Moved the creation of the tags down, since if we aren't pushing we likely don't want local tags either